config/install/field.storage.node.field_passen.yml

```diff
@@ -0,0 +1,19 @@
+langcode: en
+status: true
+dependencies:
+  module:
+    - node
+id: node.field_passen
+field_name: field_passen
```

Comment:
This machine name doesn't match the content model. Change here or in the content model so they match.
![starships_field_passengers_machine_name](https://github.com/notS0Solo/swapimigration/assets/1434979/12d8c90e-4106-46c8-8e74-737f25025251)

Reply:
Fixed